src/Session.php

```diff
@@ -940,6 +940,7 @@ public static function isInventory(): bool
         return (isset($_SESSION["glpiinventoryuserrunning"])
               && (
                   strpos($_SERVER['PHP_SELF'], '/inventory.php') !== false
+                  // TODO: how to check current route here?
```

Comment:
@cedric-anne @Pierstoval any hint on this one?

Reply:
Instead of checking the current route, maybe you can just define a static variable value just before handling the inventory request, in your `InventoryController`. This static variable would dissapear at the end of the request execution, so it would not perist in the session and it would not be necessary to check if the requested endpoint is the expected one.

Reply:
OK, I'll add a commit trying to do that